tests/lax_scipy_special_functions_test.py

```diff
@@ -261,5 +266,8 @@ def testBetaParameterDeprecation(self):
         lsp_special.beta(b=1, y=1)
 
 
+# softmax and log_softmax are tested in tests/nn_test.py.
```

Comment:
I think you can remove this comment. Most functions here are tested via `op_record`, so we don't need to call out that they're not tested via other means.